jobs/aws/eks-anywhere-build-tooling/eks-anywhere-attribution-periodics-release-0.12.yaml

```diff
@@ -11,70 +11,76 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+################################################################################
+# IMPORTANT: This Prowjob was generated by running `make prowjobs -C templater`.
+# DO NOT EDIT THIS FILE BY HAND! Refer to docs/prowjobs.md for instructions on
+# how to add a new Prowjob or update an existing Prowjob.
+################################################################################
 
 periodics:
-  - name: eks-anywhere-attribution-periodic-release-0-12
-    # Runs every weekday (M-F) at 1AM PDT
-    cron: "0 8 * * 1-5"
-    cluster: "prow-postsubmits-cluster"
-    decoration_config:
-      timeout: 4h
-      gcs_configuration:
-        bucket: s3://prowdataclusterstack-316434458-prowbucket7c73355c-1n9f9v93wpjcm
-        path_strategy: explicit
-      s3_credentials_secret: s3-credentials
-    extra_refs:
-    - org: aws
-      repo: eks-anywhere-build-tooling
-      base_ref: release-0.12
-    labels:
-      image-build: "true"
-    spec:
-      nodeSelector:
-        arch: "AMD64"
-      serviceaccountName: postsubmits-build-account
-      automountServiceAccountToken: false
-      containers:
-      - name: build-container
-        image: public.ecr.aws/eks-distro-build-tooling/builder-base:standard-8026816b32209021fe189380fb51e25690f08d37.2
-        env:
-        - name: PULL_BASE_REF
-          value: release-0.12
-        command:
-        - bash
-        - -c
-        - >
-          trap 'touch /status/done && rm -rf /home/prow/go/src/' EXIT
-          &&
-          build/lib/buildkit_check.sh
-          &&
-          make update-attribution-files
-        resources:
-          requests:
-            memory: "16Gi"
-            cpu: "5"
-        volumeMounts:
-        - name: ssh-auth
-          mountPath: /secrets/ssh-secrets
-          readOnly: true
-        - name: github-auth
-          mountPath: /secrets/github-secrets
-          readOnly: true
-      - name: buildkitd
-        image: moby/buildkit:v0.10.5-rootless
-        command:
-        - sh
-        args:
-        - /script/entrypoint.sh
-        securityContext:
-          runAsUser: 1000
-          runAsGroup: 1000
-      volumes:
+- name: eks-anywhere-attribution-periodic-release-0-12
+  cron: "0 8 * * 1-5"
+  cluster: "prow-postsubmits-cluster"
+  error_on_eviction: true
```

Comment:
This looks new. Do we want this set?

Reply:
i dont think this is new, maybe renamed or something? but this job should have already existed